packages/gatsby-plugin-gatsby-cloud/src/components/buttons/GatsbyIndicatorButton.js

```diff
@@ -1,40 +1,32 @@
 import React from "react"
 import IndicatorButton from "./IndicatorButton"
-import {
-  BuildErrorTooltipContent,
-  BuildSuccessTooltipContent,
-} from "../tooltips"
 import { gatsbyIcon } from "../icons"
+import { BuildStatus } from "../../models/enums"
 
-const getButtonProps = ({
-  buildStatus,
-  orgId,
-  siteId,
-  buildId,
-  isOnPrettyUrl,
-  sitePrefix,
-  erroredBuildId,
-}) => {
-  switch (buildStatus) {
-    case `BUILDING`:
-    case `ERROR`:
-    case `SUCCESS`:
-    case `UPTODATE`:
-    default: {
-      return {
-        active: true,
-      }
-    }
+const getButtonProps = ({ buildStatus, buttonIndex }) => {
+  const baseProps = {
+    testId: `gatsby`,
+    buttonIndex,
+    iconSvg: gatsbyIcon,
   }
-}
 
-export default function GatsbyIndicatorButton(props) {
-  return (
-    <IndicatorButton
-      testId="gatsby"
-      iconSvg={gatsbyIcon}
-      {...getButtonProps(props)}
-      buttonIndex={props.buttonIndex}
-    />
-  )
+  const propsByBuildStatus = {
+    [BuildStatus.SUCCESS]: null,
+    [BuildStatus.ERROR]: null,
+    [BuildStatus.BUILDING]: null,
+    [BuildStatus.UPTODATE]: {
+      active: true,
```

Comment:
This.. will actually break thing si think. we need to change it to always returning 

```
return {
        active: true,
      }
```